specification/netapp/resource-manager/readme.java.md

```diff
@@ -24,6 +25,18 @@ batch:
   - tag: package-netapp-2019-05-01
   - tag: package-2017-08-15
 ```
+### Tag: package-netapp-2020-02-01 and java
+
+These settings apply only when `--tag=package-netapp-2020-02-01 --java` is specified on the command line.
+Please also specify the `--azure-libraries-for-java-folder=<path to the root directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(tag) == 'package-netapp-2020-02-01' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.netapp.v2020_02_01
+  output-folder: $(azure-libraries-for-java-folder)/sdk/netapp/mgmt-v2020_02_01
+regenerate-manager: true
+generate-interface: true
+```
 
```

Comment:
+ @weidongxu-microsoft  to take a look this change for Java SDK impact

Reply:
LGTM

It is same as the last one `### Tag: package-netapp-2019-11-01 and java`, with change on version. So generally it is good.